### DISCUSSION TITLE
Micronaut Docker-Native Build is Broken in 4.x ?  GLIBC_2.34' not found.

### DISCUSSION THREAD

**Discussion by subash89** [micronaut-projects/micronaut-core#9691] [Q&A]
Hi, I can get this working in 3.9.x. For example as in https://github.com/ivangfr/graalvm-quarkus-micronaut-springboot/tree/master/simple-api/micronaut-simple-api

But if I migrate the above app to micronaut 4.0.3, and  when try to build native docker image, I am getting following error when starting the docker container. 

- When execute command mvn clean package -Pgraalvm -Dpackaging=docker-native 
- Build competes successfully.
- Then at running the container  docker run --name simpleapi ivanfranchin/micronaut-simple-api-jvm:1.0.0

**/app/application: /usr/glibc-compat/lib/libc.so.6: version `GLIBC_2.34' not found (required by /app/application)**

**Comment by alvarosanchez** [comment]
This is being addressed in https://github.com/micronaut-projects/micronaut-maven-plugin/pull/822
